### PR TITLE
Swagger and Postman documentation

### PR DESCRIPTION
**Pull request by learnjavawithreshma**
Created Swagger and Postman documents.

### CODE REVIEW

File: userguide/tutorials/swagger_documentation.adoc [hunk 1]
```diff
@@ -0,0 +1,111 @@
+= Swagger Documentation
+
+Kill Bill provides in-built support for http://swagger.io/swagger-ui/[Swagger UI]. Swagger UI allows viewing and interacting with Kill Bill's API without having to write any code. This can be very useful for debugging. This document is aimed at providing a detailed walkthrough of how to set up and use Kill Bill's Swagger documentation.
```
Comment (pierre) [inline userguide/tutorials/swagger_documentation.adoc:3]: Typo: `built-in`

File: userguide/tutorials/swagger_documentation.adoc [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+== View API Methods
+
+In order to view Kill Bill API methods in Swagger UI, you need to do the following:
+
+. Ensure that Kill Bill is running (either on https://docs.killbill.io/latest/getting_started.html#_aws_one_click[AWS] or via https://docs.killbill.io/latest/getting_started.html#_docker[Docker] or in https://docs.killbill.io/latest/getting_started.html#_tomcat[Tomcat]) or https://docs.killbill.io/latest/development.html#_running_the_application[standalone]
+
+. Type *<KILL_BILL_HOME>/api.html* in a browser. (Replace *<KILL_BILL_HOME>* with your Kill Bill home URL, so if Kill Bill is deployed at 127.0.0.1:8080, you can use \http://127.0.0.1:8080/api.html). This displays all Kill Bill's API methods as follows:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/swagger_ui_home.png[align=center]
+
+. You can expand any API method to know more about it simply by clicking it. For example, to retrieve all the payment methods related to an account you can click on `1/1.0/kb/accounts/{accountId}/paymentMethods` which displays the following:
```
Comment (pierre) [inline userguide/tutorials/swagger_documentation.adoc:22]: Typo: `1/1.0/kb/...`, unless I'm missing something? (Same remarks below)

File: userguide/tutorials/swagger_documentation.adoc [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+* `X-Killbill-Reason` - The reason for creating the request. Optional, can be left blank.
+
+* `X-Killbill-Comment` - Additional comment. Optional, can be left blank.
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/trigger_payment_execution2.png[align=center]
+
+. Click *Execute*. This displays the following screen:
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/swagger/trigger_payment_execution_output.png[align=center]
+
+So, you can see that a purchase payment is triggered against the default payment method for the specified `accountId`
```
Comment (pierre) [inline userguide/tutorials/swagger_documentation.adoc:109]: Trailing dot

**Review by pierre — APPROVED** [review]
Looks great! 🎉 Merging so I can see the pics in context, but I had a couple of comments.

File: userguide/tutorials/swagger_documentation.adoc (hunk 2 above)
Comment (learnjavawithreshma) [reply userguide/tutorials/swagger_documentation.adoc:22]: Do you mean in the `http://127.0.0.1:8080/api.html` URL? This works fine for me without `1/1.0/kb/...`

File: userguide/tutorials/swagger_documentation.adoc (hunk 1 above)
Comment (learnjavawithreshma) [reply userguide/tutorials/swagger_documentation.adoc:3]: Fixed

File: userguide/tutorials/swagger_documentation.adoc (hunk 3 above)
Comment (learnjavawithreshma) [reply userguide/tutorials/swagger_documentation.adoc:109]: Don't seem to find the trailing dot :), can you please let me know the line number?

File: userguide/tutorials/swagger_documentation.adoc (hunk 2 above)
Comment (pierre) [reply userguide/tutorials/swagger_documentation.adoc:22]: I was referring to `1/1.0/kb/accounts/{accountId}/paymentMethods` -- shouldn't it be `/1.0/kb/accounts/{accountId}/paymentMethods` (without the leading `1`)? It's the same below.

File: userguide/tutorials/swagger_documentation.adoc (hunk 3 above)
Comment (pierre) [reply userguide/tutorials/swagger_documentation.adoc:109]: Sorry -- _missing_ trailing dot :-)

File: userguide/tutorials/swagger_documentation.adoc (hunk 2 above)
Comment (learnjavawithreshma) [reply userguide/tutorials/swagger_documentation.adoc:22]: Fixed

**Pull request merged**